### PR TITLE
Refactor multi file handling

### PR DESCRIPTION
**Pull request by SebastianSchmidl**

### CODE REVIEW

File: cse/reader/CommentReader.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        return self
+
+
+    def close(self):
+        self.__commentsFile.close()
+        self.__articlesReader.close()
+
+
+    def __parseIterRow(self, row):
+        commentId = row[0]
```
Comment (Benedikt1992) [inline cse/reader/CommentReader.py]: `commentId = int(row[0])`

File: cse/writer/ArticleMappingWriter.py [hunk 2]
```diff
@@ -37,16 +40,16 @@ def printHeader(self, template=None):
...
 
 
-    def printData(self, data):
-        for i, articleUrl in enumerate(data):
-            self.__writer.writerow([
-                str(i),
-                articleUrl,
-            ])
-        self.__file.flush()
+    def mapToId(self, origArticleId, articleUrl):
```
Comment (Benedikt1992) [inline cse/writer/ArticleMappingWriter.py:45]: Do you really want to rely on the `origArticleId` ? (Keep in mind the old API has no original id))

Comment (SebastianSchmidl) [reply cse/writer/ArticleMappingWriter.py:45]: for missing ids, i replaced them with the url, but we can change it back to only considering the urls

Comment (Benedikt1992) [reply cse/writer/ArticleMappingWriter.py:45]: It's up to you to change it or leave it. I think we won't run in problem because of this

**Review by Benedikt1992 — APPROVED** [review]

File: cse/writer/ArticleMappingWriter.py (hunk 2 above)
Comment (SebastianSchmidl) [reply cse/writer/ArticleMappingWriter.py:45]: Keep it. 😄

**Pull request merged**